cfgov/privacy/forms.py

```diff
@@ -0,0 +1,212 @@
+        self.limit_number_of_files(uploaded_files)
+
+    # Email message
+    def send_email(self):
+        uploaded_files = self.files.getlist('supporting_documentation')
+        data = self.cleaned_data
+        data.update({'uploaded_files': uploaded_files})
+        subject = self.format_subject(data['requestor_name'])
+        from_email = settings.DEFAULT_FROM_EMAIL
+        recipient_list = ['FOIA@consumerfinance.gov']
+
+        body = self.email_body(data)
+
+        email = EmailMessage(
+            subject,
+            body,
+            from_email,
+            recipient_list,
+            reply_to=[data['requestor_email']]
+        )
+        email.content_subtype = 'html'
+
+        for f in uploaded_files:
+            email.attach(f.name, f.read(), f.content_type)
+
+        try:
+            email.send()
+        except BadHeaderError:
+            return HttpResponse('Invalid header found.')
+
+
+class DisclosureConsentForm(PrivacyActForm):
+    consent = forms.BooleanField(
+        widget=forms.CheckboxInput(attrs={'class': 'a-checkbox'}),
+    )
+    # Additional fields beyond what's defined in PrivacyActForm
+    recipient_name = forms.CharField(
+        label='Name of recipient',
+        widget=forms.TextInput(attrs=text_input_attrs),
+    )
+    recipient_email = forms.EmailField(
+        label='Email address',
+        widget=forms.EmailInput(attrs=text_input_attrs),
+    )
+
+    def format_subject(self, name):
+        return f'Disclosure request from consumerfinance.gov: {name}'
+
+    email_template = 'privacy/disclosure_consent_email.html'
+
+    def email_body(self, data):
+        num_files = len(data['uploaded_files'])
+        data.update({'num_files': num_files,
+                     'consent_text': self.consent_text})
+        return loader.render_to_string(self.email_template, data)
+
+
+class RecordsAccessForm(PrivacyActForm):
+    # Inherit form fields from the PrivacyActForm class
+    consent_text = '''
+        I declare under penalty of perjury under the laws of the United
+        States of America that the foregoing is true and correct, and that
+        I am the person named above and requesting access to my records
+        [or records that I am entitled to request as the parent of a minor
+        or the legal guardian of an incompetent], and I understand that
+        any falsification of this statement is punishable under the
+        provisions of 18 U.S.C. § 1001 by a fine, imprisonment of not
+        more than five years, or both, and that requesting or obtaining
+        any record(s) under false pretenses is punishable under the
+        provisions of 5 U.S.C. § 552a(i)(3) by a fine of not more than
+        $5,000.'''
+    consent = forms.BooleanField(
+        label=consent_text,
+        widget=forms.CheckboxInput(attrs={'class': 'a-checkbox'}),
+    )
+
+    def format_subject(self, name):
+        return f'Records request from consumerfinance.gov: {name}'
```

Comment:
I notice that if you include a unicode character in the name, it gets encoded weirdly in the console output. For example if I make the name "é" (%E9), the output looks like:
 
```
Subject: =?utf-8?q?Records_request_from_consumerfinance=2Egov=3A_=C3=A9?=
```

Reply:
Unicode characters appear to render correctly in the actual email output, so I think this is just an issue in dev logs.